### ISSUE TITLE
What file to use to diff the reasoned changes

### ISSUE THREAD

**Issue by manulera** [pombase/fypo#4202]
Hi @matentzn 

I have made a github action to post the asserted changes that we add to fypo-edit on pull request to master to be able to review the changes and to include this in each release.

However, it would be nice to include the differences in reasoned changes in the release. What file would be the right one for this? I have tried with `fypo.owl`, but then it shows me all the changes from other imported ontologies as well, which is not what I would want.

TL;DR: Is there a place where the difference between fypo, fypo-full, fypo-simple, etc. are explained?

**Comment by matentzn** [comment]
See https://oboacademy.github.io/obook/reference/release-artefacts/

you can use fypo-simple, should be good enough.

**Comment by manulera** [comment]
Alright, thanks @matentzn I will look into it for the next release then.
